.github/workflows/cibuild.yml

```diff
@@ -43,18 +41,12 @@ jobs:
           popd
       - name: Install
         run: |
-          source $(conda info --base)/etc/profile.d/conda.sh
-          conda activate pytaxonkit
```

Comment:
Using the login shell (`bash -l`) ensures the environment is loaded automatically.